src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -72,6 +75,8 @@ object Boot extends App {
       ldapConfig.getString("userDnFormat")
     )
 
+    enableServiceAccount(gcsDAO, userDirDAO)
+
```

Comment:
automatically enable in ldap the service account user so we don't need to remember to do it in each environment